src/scripts/prettier/.prettierrc.js

```diff
@@ -1,5 +1,4 @@
 module.exports = {
   printWidth: 100,
-  parser: 'typescript',
```

Comment:
are you sure this is not needed? Have you tried running prettier from CLI?

Reply:
yes, I tried it on lexdoit and it's working greate (lexdoit has almost everything: js, jsx, ts, tsx and scss files)